### DISCUSSION TITLE
[firebase_analytics] Support Windows platform

### DISCUSSION THREAD

**Discussion by p1318k** [firebase/flutterfire#11280] [Feature request]
"firebase_core" package supports Windows. So, It may possible support windows on "firebase_analytics".
I look into the "firebase-cpp-sdk", but is doesn't support Google Analytics.

Is there any plan or progress for support firebase analytics on Windows platform?

**Comment by andynvt** [comment]
up please

**Comment by o-glethorpe** [comment]
Guys is there a roadmap for windows support?
